### ISSUE TITLE
Support HAR import

### ISSUE THREAD

**Issue by jarrodek** [jarrodek/ChromeRestClient#194]
Originally reported on Google Code with ID 197
```
Feature suggestion: allow to open a request from a HAR/curl format.

Since Chrome Developer Tools allows to copy the request from the Network tab on the
HAR/curl format, it would be nice that the application can directly import a request
parsing that format, making the replaying of a specific request easier.

Thanks!
```

Reported by `AlphaGma` on 2013-08-19 16:50:24

**Comment by jarrodek** [comment]
It make sense (since Chrome Dev Tolls use it) and it will be implemented.

**Comment by jarrodek** [comment]
This should be fixed with #178 as well
